src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -99,6 +104,21 @@ private ResponseEntity postEventInternal(final String eventTypeName,
         }
     }
 
+    private void reportMetrics(final EventTypeMetrics eventTypeMetrics, final EventPublishResult result,
+                               final String eventsAsString, final int eventCount) {
+        if (result.getStatus() == EventPublishingStatus.SUBMITTED) {
+            eventTypeMetrics.reportSizing(eventCount, eventsAsString.getBytes(StandardCharsets.UTF_8).length - 2);
```

Comment:
@lmontrieux I see, you're right. But it's already an improvement over what we have and i think it's not worth the effort of solving it with 100% correctness.